Backend/EmailTemplate/EmailMessages.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.IO;
+using FRITeam.Swapify.Backend.Notification;
+using FRITeam.Swapify.Entities.Enums;
+
+namespace FRITeam.Swapify.Backend.EmailTemplate
+{
+    class EmailMessages
+    {
+
+        public Email Mail { get; set; }
+
+
+        public void SetSubject(TypeOfMessage messageType)
+        {
+            if (messageType == TypeOfMessage.RegistrationMessage)
+                Mail.Subject = "Welcome on Swapify";
+            else if (messageType == TypeOfMessage.ConfirmSwapMessage)
+                Mail.Subject = "We found student which want to swap with you";
+            else if (messageType == TypeOfMessage.ForgotPasswordMessage)
+                Mail.Subject = "Password change request";
+            else if (messageType == TypeOfMessage.CreateSwapRequest)
+                Mail.Subject = "Swap request";
+        }
+
+        public void SetBody(TypeOfMessage messageType)
+        {
+            using (StreamReader reader = new StreamReader("EmailTemplate.html"))
+            {
+                Mail.Body = reader.ReadToEnd();
+            }
+
+            string EmailBody = String.Empty;
+
+            if (messageType == TypeOfMessage.RegistrationMessage)
+            {
+                EmailBody = "Your registration was successful, now you can edit your timetable and find other" +
```

Comment:
I would prefer multiple HTML templates (one for each TypeOfMessage), that already contain e-mail body...
You could than fill all necessary values using Mustache/Handlebars, as I told you few weeks before